app/forms/schools/add_participant_form.rb

```diff
@@ -120,12 +121,19 @@ def creators
     end
 
     def save!
-      creators[participant_type].call(
-        full_name: full_name,
-        email: email,
-        school_cohort: school_cohort,
-        mentor_profile_id: mentor&.mentor_profile&.id,
-      )
+      ActiveRecord::Base.transaction do
+        profile = creators[participant_type].call(
+          full_name: full_name,
+          email: email,
+          school_cohort: school_cohort,
+          mentor_profile_id: mentor&.mentor_profile&.id,
+        )
+
+        ParticipantMailer.participant_added(participant_profile: profile).deliver_later
+        profile.update_column(:request_for_details_sent_at, Time.zone.now)
```

Comment:
Question - should we put this in the mailer? On one hand, mailer traditionally does not change the state, on the other this column exists in precisely this purpose so it would avoid a bit of duplication around the code?